src/__snapshots__/geoclient.test.js.snap

```diff
@@ -1454,6 +1262,6 @@ New York, NY 10007",
   },
   "lat": 40.7128,
   "long": -74.006,
-  "valid": true,
+  "valid": false,
```

Comment:
This is probably a clue as to how this test has changed. Note that the lat/lng values above have barely changed, but the formatted_address is in India rather than the Financial District